src/test/java/tests/FilterEvalTests.java

```diff
@@ -124,6 +124,24 @@ public void in() {
         assertEquals(false, matches("in:something a is", issue));
     }
 
+    /**
+     * Helper method to test the parsing of the milestone qualifier and its values
+     * Uses the relevant milestone qualifier string and the stub issue and model as parameters
+     */
+    private void milestoneTest(String milestoneQualifier, TurboIssue issue, IModel model) {
```

Comment:
Instead of the comment, you can name the method `testMilestoneParsing` or something like that? method should be named as actions not nouns.